### DISCUSSION TITLE
Make next videos pullable up

### DISCUSSION THREAD

**Discussion by Turhvjbufv** [anddea/revanced-patches#476] [Q&A] [answered]
![Screenshot_20240516-221039_ReVanced Extended_1.png](https://github.com/anddea/revanced-patches/assets/91851160/544de649-94e9-4a37-b214-6a6e297ef387)
Currently pulling up does nothing, I want it to behave how it usually goes and pull up the videos 
Can't seem to find it in settings tho I'm sure it's an option

**Comment by anddea** [comment]
By default, you can pull up for related videos, if you clean install the app. The option is in Player > Fullscreen.

**Reply by Turhvjbufv** [reply]
![Screenshot_20240516-223047_ReVanced Extended_1.png](https://github.com/anddea/revanced-patches/assets/91851160/2bc47a73-8dfc-4642-b6e0-aba32f6219ff)

This option? 
~~Doing a clean install is annoying since I'm on root version and I also change a lot of settings~~

**Reply by anddea** [reply]
Player > Fullscreen > Hide related video overlay

**Reply by Turhvjbufv** [reply, marked as answer]
Couldn't find it, tnx anyway tho
Asked in rvx chat and here's the answer
full screen> disable "hide end screen overlay" & "hide auto play preview container "

**Reply by anddea** [reply]
I guess, you're using older version.
